app/views/cryptids/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1><%= @cryptid.name %></h1>
 <h2> <%= @cryptid.category.name %> - <%= @cryptid.region.name %> </h2>
 
-<img src= <%= @cryptid.pic_url %> alt= <%= @cryptid.name %>>
+<img id="cryptid_show_pic" src= <%= @cryptid.pic_url %> alt= <%= @cryptid.name %>>
```

Comment:
Per convention I'd love to see your id separated with dashes instead of underscores: `cryptid-show-pic`